pages/[testament]/index.js

```diff
@@ -145,4 +83,31 @@ export default function Page() {
       </main>
     </>
   );
-}
\ No newline at end of file
+}
+
+export async function getStaticPaths() {
```

Comment:
Why do these two functions need to be exports?

Also put a comment above each of the two explaining what their purpose is. Like what do they fetch for us?